docs/contribute/development-environment.md

```diff
@@ -12,10 +12,10 @@ If you want to execute the STF tests it is recommended to use eclipse. Otherwise
 * Install [GEF Legacy](https://projects.eclipse.org/projects/tools.gef/downloads) into Eclipse
 * You need an **Intellij IDEA** installation which is used for dependency resolution. Install a current version of [**IntelliJ IDEA**](https://www.jetbrains.com/idea/download/#section=linux) (we have only tested Saros/I for IntelliJ releases 2017.X and later)
 
-Set the **system-wide environment variable `ECLIPSE_HOME`** to the eclipse installation dir that contains the directory `plugins`.
+Set the **system-wide environment variable `ECLIPSE_HOME`** to the eclipse installation dir that contains the directory `plugins`.  
```

Comment:
Remove trailing whitespace? Same goes for line 18.

Reply:
Trailing whitespaces are a possible solution for a line break in md. I found a [discussion of this topic](https://gist.github.com/shaunlebron/746476e6e7a4d698b373) with a few other alternatives. I think using a backslash instead of the whitespaces is more understandable :wink: I will fix this.

Reply:
The backslash does no work in jekyll. Therefore I will use `<br/>` which works in github and jekyll.